drivers/flash/Kconfig.nxp_s32

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config FLASH_NXP_S32_QSPI_NOR
+	bool "NXP S32 QSPI NOR driver"
+	default y
+	depends on DT_HAS_NXP_S32_QSPI_NOR_ENABLED
+	select MEMC
+	select MEMC_NXP_S32_QSPI
```

Comment:
Should this selection be neccesary? I would expect the memc controller having `status="okay"` in devicetree to enable this Kconfig

Reply:
right, removed, it will get selected by enabling the `qspi` controller node. Btw, this is also present in https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/flash/Kconfig.mcux#L61-L62